stan/math/prim/scal/prob/beta_binomial_lcdf.hpp

```diff
@@ -103,10 +103,9 @@ namespace stan {
         const T_partials_return mu = alpha_dbl + n_dbl + 1;
         const T_partials_return nu = beta_dbl + N_dbl - n_dbl - 1;
 
-        const T_partials_return F = F32((T_partials_return)1, mu,
-                                                    -N_dbl + n_dbl + 1,
-                                                    n_dbl + 2, 1 - nu,
-                                                    (T_partials_return)1);
+        T_partials_return F;
+        F = F32((T_partials_return)1, mu, -N_dbl + n_dbl + 1, n_dbl + 2,
```

Comment:
Yeah, I think it's a bigger problem, should be a separate PR, I saw a few places that seem like they do over-promotion (?)